### PR TITLE
Expose ElementArray as type to the WebdriverIO namespace

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

Given that we have `Element` within the WebdriverIO global namespace I suggest to do the same for `ElementArray` given it can be used various times within the test and this would safe a lot of imports.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

refs: webdriverio/webdriverio#6502

### CODE REVIEW

File: tests/typings/sync/sync.ts [hunk 1]
```diff
@@ -23,6 +23,9 @@ nsBrowser.clearMockCalls('')
 const nsElem: WebdriverIO.Element = {} as any
 nsElem.click()
 
+const nsElems: WebdriverIO.ElementArray = {} as any
+nsElems.foundWith.toUpperCase()
```
Comment (erwinheitzman) [inline tests/typings/sync/sync.ts:27]: why is this line added here?

**Review by erwinheitzman — COMMENTED** [review]
small question before approval

File: tests/typings/sync/sync.ts (hunk 1 above)
Comment (christian-bromann) [reply tests/typings/sync/sync.ts:27]: to test the `ElementArray` typings

**Review by erwinheitzman — APPROVED** [review]
